genie-client/build.gradle

```diff
@@ -1,9 +1,12 @@
+apply plugin: 'groovy'
+
 dependencies {
     /*******************************
      * Compile Dependencies
      *******************************/
 
     compile project(":genie-common")
+    compile "org.codehaus.groovy:groovy:2.3.3"
```

Comment:
@liviutudor Can you follow the pattern of putting this version into ```gradle.properties``` and then reusing it as a variable? We'd like to keep the versions for things in one place so we don't have to hunt through all the build files for them.